### PR TITLE
Fix 149 Outdated content and links in chapter 4.11

### PR DESCRIPTION
**Pull request by lukaszlu**
This PR covers issue #149 .

- [X] This PR handles the issue and requires no additional PRs.
- [X] You have validated the need for this change.

**What did this PR accomplish?**

- Removed obsolete links and resources
- Fixed not working links
- Listed new tools that replace obsolete tools

Thank you for your contribution!

### CODE REVIEW

File: document/4_Web_Application_Security_Testing/4.11_Business_Logic_Testing/4.11_Testing_for_Business_Logic.md [hunk 1]
```diff
@@ -100,39 +100,38 @@ To view and modify HTTP/HTTPS headers, post parameters, and observe the DOM of t
...
 
 Request Maker is a tool for penetration testing. With it you can easily capture requests made by web pages, tamper with the URL, headers and POST data and, of course, make new requests
 
-- [Cookie Editor for Chrome](https://chrome.google.com/webstore/detail/fngmhnnpilhplaeedifhccceomclgfbg?hl=en-US)
-- [Cookie Editor for Firefox](https://addons.mozilla.org/en-US/firefox/addon/http-request-maker)
+- [Cookie Editor for Chrome](https://chrome.google.com/webstore/detail/fngmhnnpilhplaeedifhccceomclgfbg)
+- [Cookie Editor for Firefox](https://addons.mozilla.org/en-US/firefox/addon/cookie-editor)
 
-Edit This Cookie is a cookie manager. You can add, delete, edit, search, protect and block cookies
+Cookie Editor is a cookie manager. You can add, delete, edit, search, protect and block cookies
```
Comment (kingthorin) [inline document/4_Web_Application_Security_Testing/4.11_Business_Logic_Testing/4.11_Testing_for_Business_Logic.md]: Are "protect and block" supposed to be a single entity or should there be a serial/oxford comma here?

File: document/4_Web_Application_Security_Testing/4.11_Business_Logic_Testing/4.11_Testing_for_Business_Logic.md [hunk 2]
```diff
@@ -93,47 +93,45 @@ To view and modify HTTP/HTTPS headers, post parameters, and observe the DOM of t
...
 
-- [HTTP Response Browser](https://chrome.google.com/webstore/detail/mgekankhbggjkjpcbhacjgflbacnpljm?hl=en-US)
+- [Cookie Swap for Chrome](https://chrome.google.com/webstore/detail/dffhipnliikkblkhpjapbecpmoilcama)
 
-Make HTTP requests from your browser and browse the response (HTTP headers and source). Send HTTP method, headers and body using XMLHttpRequest from your browser then view the HTTP status, headers and source. Click links in the headers or body to issue new requests. This plug-in formats XML responses and uses [Syntax Highlighter](http://alexgorbatchev.com/SyntaxHighlighter/)
+Swap My Cookies is a session manager, it manages your cookies, letting you login on any website with several different accounts. You can finally login into Gmail, yahoo, hotmail, and just any website you use, with all your accounts; if you want to use another account just swap profile!
 
-- [Firebug lite for Chrome](https://chrome.google.com/webstore/detail/bmagokdooijbeehmkpknfglimnifench)
+- [Firefox Multi-Account Containers](https://addons.mozilla.org/en-US/firefox/addon/multi-account-containers)
```
Comment (ThunderSon) [inline document/4_Web_Application_Security_Testing/4.11_Business_Logic_Testing/4.11_Testing_for_Business_Logic.md]: Firefox currently has that functionality built-in. Should this still be recommended? If you try to open any link into a new tab, you can choose to open it in a containerized tab.

File: document/4_Web_Application_Security_Testing/4.11_Business_Logic_Testing/4.11_Testing_for_Business_Logic.md [hunk 3]
```diff
@@ -93,47 +93,45 @@ To view and modify HTTP/HTTPS headers, post parameters, and observe the DOM of t
...
-- [Cookie Editor for Firefox](https://addons.mozilla.org/en-US/firefox/addon/http-request-maker)
+- [Cookie Editor for Chrome](https://chrome.google.com/webstore/detail/fngmhnnpilhplaeedifhccceomclgfbg)
+- [Cookie Editor for Firefox](https://addons.mozilla.org/en-US/firefox/addon/cookie-editor)
 
-Edit This Cookie is a cookie manager. You can add, delete, edit, search, protect and block cookies
+Cookie Editor is a cookie manager. You can add, delete, edit, search, protect and block cookies
 
-- [Session Manager for Chrome](hhttps://chrome.google.com/webstore/detail/session-manager/mghenlmbmjcpehccoangkdpagbcbkdpc)
-- [Session Manager for Firefox](https://addons.mozilla.org/en-US/firefox/addon/http-request-maker)
+- [Session Manager for Chrome](https://chrome.google.com/webstore/detail/session-manager/mghenlmbmjcpehccoangkdpagbcbkdpc)
```
Comment (ThunderSon) [inline document/4_Web_Application_Security_Testing/4.11_Business_Logic_Testing/4.11_Testing_for_Business_Logic.md]: Last updated 2016

File: document/4_Web_Application_Security_Testing/4.11_Business_Logic_Testing/4.11_Testing_for_Business_Logic.md [hunk 4]
```diff
@@ -93,47 +93,45 @@ To view and modify HTTP/HTTPS headers, post parameters, and observe the DOM of t
...
-- [Cookie Swap for Chrome](https://chrome.google.com/webstore/detail/dffhipnliikkblkhpjapbecpmoilcama?hl=en-US)
-- [Cookie Swap for Firefox](https://addons.mozilla.org/en-US/firefox/addon/http-request-maker)
+- [MySessions for Firefox](https://addons.mozilla.org/en-US/firefox/addon/my-sessions/)
 
-Swap My Cookies is a session manager, it manages your cookies, letting you login on any website with several different accounts. You can finally login into Gmail, yahoo, hotmail, and just any website you use, with all your accounts; if you want to use another account just swap profile!
+MySessions is a simple session manager. You can quickly save your current browser state and reload it whenever necessary.
+This is not a replacement for Michael Kraft's "Session Manager", but it allows you to import saved sessions from the "Session Manager".
 
-- [HTTP Response Browser](https://chrome.google.com/webstore/detail/mgekankhbggjkjpcbhacjgflbacnpljm?hl=en-US)
+- [Cookie Swap for Chrome](https://chrome.google.com/webstore/detail/dffhipnliikkblkhpjapbecpmoilcama)
```
Comment (ThunderSon) [inline document/4_Web_Application_Security_Testing/4.11_Business_Logic_Testing/4.11_Testing_for_Business_Logic.md]: Last updated 2013. Should this really be recommended?

File: document/4_Web_Application_Security_Testing/4.11_Business_Logic_Testing/4.11_Testing_for_Business_Logic.md [hunk 5]
```diff
@@ -93,47 +93,45 @@ To view and modify HTTP/HTTPS headers, post parameters, and observe the DOM of t
...
 
-- [Cookie Swap for Chrome](https://chrome.google.com/webstore/detail/dffhipnliikkblkhpjapbecpmoilcama?hl=en-US)
-- [Cookie Swap for Firefox](https://addons.mozilla.org/en-US/firefox/addon/http-request-maker)
+- [MySessions for Firefox](https://addons.mozilla.org/en-US/firefox/addon/my-sessions/)
 
-Swap My Cookies is a session manager, it manages your cookies, letting you login on any website with several different accounts. You can finally login into Gmail, yahoo, hotmail, and just any website you use, with all your accounts; if you want to use another account just swap profile!
+MySessions is a simple session manager. You can quickly save your current browser state and reload it whenever necessary.
+This is not a replacement for Michael Kraft's "Session Manager", but it allows you to import saved sessions from the "Session Manager".
```
Comment (ThunderSon) [inline document/4_Web_Application_Security_Testing/4.11_Business_Logic_Testing/4.11_Testing_for_Business_Logic.md]: This is no longer available for use in FF Quantum

**Comment by kingthorin** [comment]
> Should we add a note for the reader to be aware of the permissions that those extensions require?

I think that’ll just become a maintenance nightmare. Plus browsers seem to do a fine job of advertising/asking at install time.

**Comment by ThunderSon** [comment]
So I believe that a small note should be set to make sure that we are not liable on any updates that occur to those extensions. What do you think @kingthorin ?

**Comment by kingthorin** [comment]
Yup, something short and generic would work.

File: document/4_Web_Application_Security_Testing/4.11_Business_Logic_Testing/4.11_Testing_for_Business_Logic.md (hunk 1 above)
Comment (lukaszlu) [reply document/4_Web_Application_Security_Testing/4.11_Business_Logic_Testing/4.11_Testing_for_Business_Logic.md]: OK, I will add the oxford comma, thank you.

File: document/4_Web_Application_Security_Testing/4.11_Business_Logic_Testing/4.11_Testing_for_Business_Logic.md (hunk 2 above)
Comment (lukaszlu) [reply document/4_Web_Application_Security_Testing/4.11_Business_Logic_Testing/4.11_Testing_for_Business_Logic.md]: I will remove this plug-in from the list.
Personally, I support the approach not to recommend any third-party plug-in tools as such selections are fleeting, impermanent.

File: document/4_Web_Application_Security_Testing/4.11_Business_Logic_Testing/4.11_Testing_for_Business_Logic.md (hunk 3 above)
Comment (lukaszlu) [reply document/4_Web_Application_Security_Testing/4.11_Business_Logic_Testing/4.11_Testing_for_Business_Logic.md]: OK, I will remove this old plug-in from the list as well. Thank you.

File: document/4_Web_Application_Security_Testing/4.11_Business_Logic_Testing/4.11_Testing_for_Business_Logic.md (hunk 4 above)
Comment (lukaszlu) [reply document/4_Web_Application_Security_Testing/4.11_Business_Logic_Testing/4.11_Testing_for_Business_Logic.md]: OK, I will remove this old plug-in from the list as well. Thank you.

File: document/4_Web_Application_Security_Testing/4.11_Business_Logic_Testing/4.11_Testing_for_Business_Logic.md (hunk 5 above)
Comment (lukaszlu) [reply document/4_Web_Application_Security_Testing/4.11_Business_Logic_Testing/4.11_Testing_for_Business_Logic.md]: OK, I will remove this old plug-in from the list as well. Thank you.

**Comment by kingthorin** [comment]
It's not really a legal statement, just a reminder. Something like:

> Remember: When installing addons you should always be diligent in considering the permissions they request and your browser usage habbits.

Or something like that. (FOr my 2 cents).

**Pull request merged**